feat: CSS Modules support (#4852)
remix-run/remix: docs/guides/migrating-react-router-app.md

**Note:** Remix does not currently support CSS processing directly. If you use preprocessors like Sass, Less, or PostCSS, you can run those as a separate process in development.

> We also do not yet support [CSS Modules][css-modules] by default, but it is currently [available behind a feature flag.][css-modules]

Reviewer: ```suggestion
We do process [CSS Modules][css-modules], but support is currently [opt-in behind a feature flag][css-modules].
```
